ui/app/pages/send/send.selectors.js

```diff
@@ -72,7 +72,11 @@ function accountsWithSendEtherInfoSelector (state) {
 }
 
 function getAddressBook (state) {
-  return state.metamask.addressBook
+  const addressBookEntries = Object.entries(state.metamask.addressBook).map(([key, account]) => {
+    return Object.assign({}, account, state.metamask.addressBook[key])
```

Comment:
Will `account` and `state.metamask.addressBook[key]` (on line 76) refer to the same object?

If so, maybe we could simplify this to:
```
const addressBookEntries = Object.entries(state.metamask.addressBook).map(([key, account]) => account)
```
Or maybe even: `const addressBookEntries = Object.values(state.metamask.addressBook)`

Reply:
And I guess this changed because the `state.metamask.addressBook` changed from an array to an object? Is that correct?

Reply:
ahh good catch, i made this change to keep the send flow working in my local testing, removing from this pr.